dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImplTest.java

```diff
@@ -330,6 +333,94 @@ public void testIsCheckInSafeWithoutRelationshipsShouldReturnTrue() {
     }
 
 
+    /**
+     * Method to test: {@link ESContentletAPIImpl#lock(Contentlet, User, boolean)}
+     * Given Scenario: A user without permission try to lock a contentlet
+     * ExpectedResult: Should throw a DotSecurityException
+     *
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test(expected = DotSecurityException.class)
+    public void whenTryToLockShouldThrowDotSecurityException() throws DotSecurityException, DotDataException {
+        final Role role = new RoleDataGen().nextPersisted();
+        final User user = new UserDataGen().roles(role).nextPersisted();
+        final ContentType contentType = new ContentTypeDataGen().nextPersisted();
+        final Contentlet contentlet = new ContentletDataGen(contentType.id()).next();
+
+        final ESContentletAPIImpl esContentletAPI = new ESContentletAPIImpl();
+        final Contentlet contentletSaved = esContentletAPI.checkin(contentlet, APILocator.systemUser(), false);
+        esContentletAPI.lock(contentletSaved, user, false);
+    }
+
+    /**
+     * Method to test: {@link ESContentletAPIImpl#lock(Contentlet, User, boolean)}
+     * Given Scenario: A user with {@link PermissionLevel#EDIT} permission try to lock a contentlet
+     * ExpectedResult: Should work
+     *
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test
+    public void whenTryToLockShouldWork() throws DotSecurityException, DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5365897)